web/src/notifications.js

```diff
@@ -619,6 +641,15 @@ export function notify_local_mixes(messages, need_user_to_scroll) {
             continue;
         }
 
+        const muted_narrow = get_muted_narrow(message);
+        if (muted_narrow) {
+            notify_unmute(muted_narrow, message.stream_id, message.topic);
+        }
+        // We don't `continue` after showing the unmute banner, allowing multiple
+        // banners (at max 2 including the unmute banner) to be shown at once,
+        // as it's common for the unmute case to occur simultaneously with
+        // another banner's case, like sending a message to another narrow.
```

Comment:
This comment belongs at the end of the `if` block, rather than outside -- i.e. in the place where one might have otherwise had a `continue` statement.

Generally, one places a comment just before the code that one is using it to explain; or in this case, though in this case the code we are explaining is the empty string :).